app/src/main/java/kth/jjve/xfran/adapters/CalendarAdapter.java

```diff
@@ -2,8 +2,7 @@
 
 /*
 Function: adapter for the weekly calendar
-Activity: CalendarViewActivity
-Jitse van Esch, Elisa Perini & Mariah Sabioni
+Activity: EventActivity
 based on: https://github.com/codeWithCal/CalendarTutorialAndroidStudio/tree/WeeklyCalendar
```

Comment:
Btw, not necessary now, but most of the stuff we got from that link has been adapted to our use quite a lot, so we might want to rethink in the future how do we credit that github